caffe2/utils/threadpool/pthreadpool_impl.cc

```diff
@@ -27,20 +27,21 @@ void pthreadpool_compute_1d(
           range);
 }
 
+void pthreadpool_parallelize_1d(
```

Comment:
Just for context, `pthreadpool_parallelize_1d` why is this added? Is there something on C2 side that uses this interface?